src/content/about.md

```diff
@@ -91,60 +91,70 @@ app                                                       [target]
+You can think of broccoli-plugins much like a simple programming language, where the output of a function can be
 passed as the input(s) to another function.
 
 E.g.:
 
 ```js
-let js = babel('src');
-js = uglify(js);
+export default (options) => {
+    let js = babel('src');
+    js = uglify(js);
 
-let css = sass('src/styles', 'site.scss');
-css = autoprefixer(css);
+    let css = sass('src/styles', 'site.scss');
+    css = autoprefixer(css);
 
-const result = merge([js, css]);
-
-module.exports = result;
+    return merge([js, css]);
+};
 ```
+
 This could also be expressed as:
 ```js
-module.exports = merge([
-    uglify(
-        babel('src')
-    ),
-    autoprefixer(
-        sass('src/styles', 'site.scss')
-    )
-]);
+export default (options) => {
+     return merge([
+        uglify(
+            babel('src')
+        ),
+        autoprefixer(
+            sass('src/styles', 'site.scss')
+        )
+    ])
+};
 ```
 
-In the above, a `src` directory is passed to the `babel()` plugin (which will convert our new ES6 syntax into 
-ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will minify our JS 
+In the above, a `src` directory is passed to the `babel()` plugin (which will convert our new ES6 syntax into
+ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will minify our JS
 into a smaller format. The output of `uglify()` will in turn be passed as 1 input to `merge()`.
 
 Additionally, a `src/styles` directory and the input file `site.scss` is passed to `sass()` which will convert your
 `.scss` files into `.css` files, its output (your css) is passed as an input to `autoprefixer()` which will add
 vendor  prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
 
-The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
-uglified JavaScript and out vendor prefixed css. This then becomes our final output and is what is written to 
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our
+uglified JavaScript and out vendor prefixed css. This then becomes our final output and is what is written to
 our target (destination) directory.
 
+Finally, the `export default () => ` line exports a function that Broccoli will invoke with an `options` object,
+that contains an `env` property to indicate the environment. This is set from the `--environment` CLI argument and
+defaults to `development`.
+
 This should all be fairly familiar to you if you've ever written JavaScript (or any programming language for that
 matter) before, it's just inputs and output.
 
+Note: [ES Modules](https://nodejs.org/api/esm.html) syntax is supported by Broccoli in the current master. Prior
+to this, standard CommonJs syntax with `require` and `module.exports` should be used.
+
```

Comment:
out of curiosity, does this basically just mean you're providing default exports now?

Reply:
The standard practice will be to use `export default` from the Brocfile. Is that what you mean?